types/index.d.ts

```diff
@@ -445,7 +445,7 @@ export interface MessageInputState {
   fileUploads: SeamlessImmutable.ImmutableObject<{ [id: string]: FileUpload }>;
   emojiPickerIsOpen: boolean;
   // ids of users mentioned in message
-  mentioned_users: string[];
```

Comment:
I think name should be an optional field. The only required field on a user is `id`. Also true for other areas. Probably want to check if a user has a name, then use it otherwise use id.

Reply:
Hmm, I see, thanks :). I didn't realise name is an optional field.

I think I might change the type of that state variable to `Client.UserResponse[]`, so it's consistent with the Message object, and it makes it easier to check if a user is still mentioned, because I'll have access to both name and id.

Reply:
Sounds good!

Reply:
I was looking into something that incidentally touches this, and I think we get `Client.User[]`, and not `Client.UserResponse[]` there, but I'm not sure. I wasn't looking into that field specifically. It would be nice to double-check, though.

Reply:
I double checked, it seems like we get `UserResponse`, both from the parent (as part of the `message` prop), and from the actual input, as part of `ChatAutoCompleteProps` which contains `users: Client.UserResponse[];`.

I just added the changes, should be all good now.